broadcast/src/buffered/engine.rs

```diff
@@ -324,14 +334,4 @@ impl<
             !waiters.is_empty()
         });
     }
-
-    /// Respond to a waiter with a message.
-    /// Increments the appropriate metric based on the result.
-    fn respond(&mut self, responder: oneshot::Sender<M>, msg: M) {
-        let result = responder.send(msg);
-        self.metrics.get.inc(match result {
```

Comment:
metrics are removed here but not re-added

Reply:
Still playing with ergonomics on this one 👍